components/brave_wallet_ui/common/hardware/ledgerjs/eth_ledger_bridge_keyring.test.ts

```diff
@@ -114,7 +114,7 @@ test('Sign personal message successfully', () => {
   ledgerHardwareKeyring.app.signature = { v: 1, r: 'b68983', s: 'r68983' }
   return expect(ledgerHardwareKeyring.signPersonalMessage(
     'm/44\'/60\'/0\'/0/0', 'message'))
-    .resolves.toStrictEqual({ payload: '0xb68983r68983-26', success: true })
+    .resolves.toStrictEqual({ payload: '0xb68983r68983-1a', success: true })
```

Comment:
When `v` is `1`, the serialised hex should be `01` and not `-1a`. Let's also have another test case where `v` is `27` or `28`.

Reply:
done